en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md

```diff
@@ -22,16 +22,20 @@ Two key performance metrics were used to measure the performance of each test.
 - Throughput: The number of requests that the WSO2 Micro Integrator processes during a specific time interval (e.g. per second).. 
 - Response Time: The end-to-end latency for an operation of invoking a service in WSO2 Micro Integrator. The complete distribution of response times was recorded.
 
-All the performance results were done on the default Micro Integrator distribution. The
-following performance results may become invalid if configurations are changed. For
-example, the default passthru io_buffer_size is 16KB. Hence the throughput significantly
-decreases for payload sizes above 16KB. If the buffer size is increased, the throughput
-might differ. Similarly when worker pool thread counts are changed, the performance of
-Micro Integrator changes significantly. Hence, for accurate results doing a performance
-test with the exact configuration and scenarios is a must.
+!!! Note
+    
+    All the performance results were done on the default Micro Integrator distribution. The
+    following performance results may become invalid if configurations are changed. For
+    example, the default passthru io_buffer_size is 16KB. Hence the throughput significantly
+    decreases for payload sizes above 16KB. If the buffer size is increased, the throughput
+    might differ. Similarly when worker pool thread counts are changed, the performance of
+    Micro Integrator changes significantly. Hence, for accurate results doing a performance
+    test with the exact configuration and scenarios is a must.
 
 ## Deployment used for the test
 
+[![MI performance test all-in-one deployment]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)
```

Comment:
```suggestion
Given below are the details of the deployment that was used for the performance test this is recorded in this documentation.

[![MI performance test all-in-one deployment]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)
```

Reply:
`this is recorded in this documentation` or `that is recorded in this documentation`

Reply:
good catch..